stable_baselines/common/base_class.py

```diff
@@ -457,7 +477,67 @@ def _save_to_file(save_path, data=None, params=None):
             cloudpickle.dump((data, params), save_path)
 
     @staticmethod
-    def _load_from_file(load_path):
+    def _save_to_file_zip(save_path, data=None, params=None):
+        """Save model to a .zip archive
+
+        :param save_path: (str or file-like) Where to store the model
```

Comment:
it also work with file like objects?

Reply:
Yes. `zipfile.ZipFile` also accepts file-likes (also tested in `test_save.py`)